web/docs/guides/database/resource-management.mdx

```diff
@@ -0,0 +1,21 @@
+---
+id: resource-management
+title: Resource Management
+description: Resource management and optimization
+---
+
+## Statement Timeouts
+
+By default, Supabase limits the maximum statement execution time to _3 seconds_ for users accessing the API using the anon key, and _8 seconds_ for authenticated users. This serves as a backstop against resource exhaustion due to either poorly written queries, or abusive usage.
```

Comment:
I think it should mention the global timeout
```suggestion
By default, Supabase limits the maximum statement execution time to _3 seconds_ for users accessing the API using the anon key, and _8 seconds_ for authenticated users, all other users are limited by 2 minutes. This serves as a backstop against resource exhaustion due to either poorly written queries, or abusive usage.
```

Reply:
Added a note